src/nvim/ex_cmds2.c

```diff
@@ -2347,11 +2363,12 @@ int do_in_runtimepath(char_u *name, int all, DoInRuntimepathCB callback,
 
           // Expand wildcards, invoke the callback for each match.
           if (gen_expand_wildcards(1, &buf, &num_files, &files,
-                                   EW_FILE) == OK) {
+                                   (flags & DIP_DIR) ? EW_DIR
+                                                     : EW_FILE) == OK) {
```

Comment:
I know the linter doesn't like this alignment of the ternary, but I find that far more readable than aligning the `:` with the `(`.
Alternatively, I could assign the value to a variable and use that in the function call.